### DISCUSSION TITLE
Compile problem about 2.0dev version

### DISCUSSION THREAD

**Discussion by ZhunyunTang** [ttadano/alamode#207] [Q&A]
Dear Dr. Tadano

I am getting the following error when installing version 2.0dev. At first I thought it was because the dependency library h5py was not installed, but after double checking I found that it had been installed successfully. In addition, I further installed hdf5, but it doesn't seem to help. I hope you can give some suggestion.

Best regards,
Tang

![error](https://github.com/ttadano/alamode/assets/93117628/2658a4ef-8bb3-43b3-b5ef-e0d73317a0e9)

**Comment by ttadano** [comment]
It seems that you tried to compile the code using a Makefile, but the Makefiles are not updated for the `2.0dev` branch. 

Please compile the code using CMake if you want to build the `2.0dev` version.

Besides, `2.0dev` is not officially supported, so please understand that I cannot answer questions related to `2.0dev`.

**Reply by ttadano** [reply]
Hi,

I've slightly modified `CMakeLists.txt` so that the hdf5 librariy is linked by default. 
(The previous version required `-DWITH_HDF5_SUPPORT=yes` for the CMake option.)

**Reply by ZhunyunTang** [reply]
That's great. I have successfully installed version 2.0dev based on the modified code.
